src/Sarif.Converters/PREFastConverter.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.IO;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+using System;
+using System.Collections.Generic;
+using System.Xml;
+using System.Xml.Serialization;
+using Microsoft.CodeAnalysis.Sarif.Converters.PREFastObjectModel;
+using System.Linq;
```

Comment:
Sort `using`s alphabetically (but with `System` `using`s first).

Reply:
Fixed

---
In reply to: [146948454](https://github.com/Microsoft/sarif-sdk/pull/657#discussion_r146948454) [](ancestors = 146948454)